uint32 rectified
uint32 ball_detected
uint32 ball_far

float32 x_ball
float32 y_ball
float32 camera_x_ball
float32 camera_y_ball